.github/workflows/docker.yaml

```diff
@@ -7,7 +7,8 @@ on:
       - master
 
 env:
-  DOCKER_IMAGE: "docker.pkg.github.com/kioui/tosti/tosti-production:latest"
+  DOCKER_IMAGE: docker.pkg.github.com/kioui/tosti/tosti:latest
+  DOCKER_TAG_PRODUCTION: docker.pkg.github.com/kioui/tosti/tosti:${{ github.sha }}
```

Comment:
If you're doing this, then probably you would just want to build a docker on every PR (with the hash as label), and build :latest on push to master.

Reply:
Yeah I thought this would be nice for downloading "older" images. If you don't agree we can remove it.

Reply:
I don't have a problem with building dockers for each PR! But this action only runs on pushes to master, so I don't see what this does

Reply:
Maybe we should tag `:stable` on release, `:latest` on push to master and `:<commithash>` on each PR?

Reply:
I made up my mind, I'll remove the commit hash push but let's use the stable and latest tags. I think that makes it pretty nice.